packages/protocol/test-sol/unit/governance/validators/Validators.t.sol

```diff
@@ -3082,6 +3128,30 @@ contract ValidatorsTest_GetMembershipInLastEpoch is ValidatorsTest {
   }
 }
 
+contract ValidatorsTest_GetTopGroupValidators is ValidatorsTest {
+  function setUp() public {
+    super.setUp();
+
+    _registerValidatorGroupWithMembersHavingSigners(group, 5);
+  }
+
+  function test_ShouldReturnTheSigner_whenL1() public {
+    address[] memory _validatorSigner = validators.getTopGroupValidators(group, 3);
+    assertEq(_validatorSigner[0], accounts.getValidatorSigner(validator));
+    assertEq(_validatorSigner[1], accounts.getValidatorSigner(vm.addr(1)));
+    assertFalse(_validatorSigner[0] == validator);
+  }
+
+  function test_ShouldReturnTheAccount_WhenL2() public {
+    _whenL2();
+    address[] memory validatorAccount = validators.getTopGroupValidators(group, 3);
+
+    assertEq(validatorAccount[0], validator);
+    assertEq(validatorAccount[1], vm.addr(1));
+    assertFalse(validatorAccount[0] == accounts.getValidatorSigner(validator));
```

Comment:
`assertNotEq `

Reply:
This is not yet available for 0.5 version. hence why I originally went with `assertFalse`